server/opts.go

```diff
@@ -380,6 +381,15 @@ func parseCluster(cm map[string]interface{}, opts *Options) error {
 			opts.Cluster.Username = auth.user
 			opts.Cluster.Password = auth.pass
 			opts.Cluster.AuthTimeout = auth.timeout
+			if auth.defaultPermissions != nil {
+				// For routes:
+				// Import is Publish
```

Comment:
Yes Import is remote publish, but more succinctly it is whether or not we will send a SUB for interest to the other side. Export is will we accept a SUB from the remote for a given subject. Both only effect interest registration IMO.

Reply:
Are you saying that I should use that instead of the above comment?

Reply:
By the way, the comment was just to explain the mapping between the existing c.perms.[pub|sub] and the route import/export...

Reply:
I think the clearer we can be the better, but your call. Not holding up LGTM.

Reply:
Got it. Made some changes there.